app/controllers/users/registrations_controller.rb

```diff
@@ -1,5 +1,12 @@
 class Users::RegistrationsController < Devise::RegistrationsController
-# before_action :configure_sign_up_params, only: [:create]
+  before_action :configure_sign_up_params, only: [:create]
+
+  def create
```

Comment:
I've never had a need to override the devise controllers before. What are we trying to account for by using our own controllers? I don't think `crapola!` is a method 😆